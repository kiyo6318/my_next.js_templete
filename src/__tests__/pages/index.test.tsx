import React from 'react';
import Home from 'src/pages/index';
import { render } from '../utils/index';

test('ホームページ', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
