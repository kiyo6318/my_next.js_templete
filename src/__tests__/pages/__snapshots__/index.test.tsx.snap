// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ホームページ 1`] = `
<div>
  <div
    class="css-0"
  >
    <h2
      class="chakra-heading css-1dklj6k"
    >
      Home
    </h2>
  </div>
</div>
`;
